roles/common/defaults/main.yml

```
@@ -28,7 +33,12 @@ ports_cfg:
     http: "{{ ports.nginx.http | default(80) }}"
     https: "{{ ports.nginx.https | default(443) }}"
   activemq:
-    openwire: "{{ ports.activemq.openwire | default(61616) }}"
+    openwire: >-
+      {% if 'external_activemq' in groups %}
```

same remark as above
```suggestion
  {% if groups.external_activemq %}
```